README.md

```diff
@@ -1 +1,69 @@
-# restli-csharp
\ No newline at end of file
+# Rest.li C#
+
+## How to Build
+
+- clone the repository locally and move into directory
+    - `git clone https://github.com/mchen07/restli-csharp.git`
+    - `cd restli-csharp`
+- run the gradle script to generate data templates:
+    - on Windows: `gradlew.bat clean build`
+    - on Mac/Linux: `./gradlew clean build`
+- open `restlicsharp.sln` in Visual Studio:
+    - build the solution
+    - run all tests for the solution to ensure everything is setup
```

Comment:
The tests currently have `http://evwillia-ld1:1338` as the URL prefix. Should I change this to something else or just document it?